web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -316,14 +307,16 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
         return chunks.join("-");
     },
 
-    _viewButtonOnClick: function(btn) {
+    _viewButtonOnClick: function(btn, clickEvent) {
 
-        btn.addClass("x-btn-selected");
+        var multiSelect = clickEvent.ctrlKey;
         var superUuid = btn.container.id.replace("fsSearchAddBtn", '');
-        this.addRecordFromSuperUuid(superUuid);
+        this.addRecordFromSuperUuid(superUuid, multiSelect);
+
+        btn.addClass("x-btn-selected");
     },
 
-    addRecordFromSuperUuid: function(superUuid) {
+    addRecordFromSuperUuid: function(superUuid, multiSelect) {
```

Comment:
The naming `super...` is a bit weird here, IMO.  It's the UUID of the metadata record, right?